File: 6kyu/sum-of-digits-slash-digital-root/old_solution.py
# https://www.codewars.com/kata/541c8630095125aba6000c00
# 6 Kyu
# Sum of Digits / Digital Root

# My Solution
def digital_root(number):
    while len(str(number)) > 1:
        number = sum(int(num) for num in str(number))
    return number


# Codewars solution 1
def digital_root(n):
    return n if n < 10 else digital_root(sum(map(int, str(n))))


# Codewars solution 2
def digital_root(n):
    return n % 9 or n and 9

File: 7kyu/is-this-a-triangle/old_solution.go
// https://www.codewars.com/kata/56606694ec01347ce800001b
// 7 kyu
// Is this a triangle?

package kata

// My Solution
func IsTriangle(a, b, c int) bool {
	return (a+b > c) && (a+c > b) && (b+c > a)
}
